net/wsconn/wsconn.go

```diff
@@ -48,10 +48,11 @@ import (
 //
 // A received StatusNormalClosure or StatusGoingAway close frame will be translated to
 // io.EOF when reading.
-func NetConn(ctx context.Context, c *websocket.Conn, msgType websocket.MessageType) net.Conn {
+func NetConn(ctx context.Context, c *websocket.Conn, msgType websocket.MessageType, remoteAddr string) net.Conn {
```

Comment:
Docs on this? Is it an IP:port always or can it be a URL sometimes too it looks like?

Reply:
Docced. It can be anything at all, because it's the r.RemoteAddr of the incoming request, which can be anything at all. For client connections, I arbitrarily decided to make it be the Websocket URL being connected to, but it wasn't deeply thought out. Just something that represents the remote endpoint, and seems somewhat sensible.